user-guide/Basic_Functionality/Protocols_and_templates/Alarm_templates/Configuring_alarm_templates/Configuring_anomaly_detection_alarms.md

```diff
@@ -74,9 +74,11 @@ To configure the Augmented Operations alarm settings for specific parameters<!--
 
    1. Select the checkbox.
 
-   1. Select the severity of the generated alarms: *Warning*, *Minor*, *Major*, or *Critical*.
+   1. Select the severity of the generated alarms.
 
-   1. Select the trend prediction range that will trigger the alarms: *Upper range (100.0 %)*, *Critical high (90.0 %)*, *Major High (80.0 %)*, or *Lower range (0.0 %)*.
+   1. Select the trend prediction range that will trigger the alarms.
+
+      The ranges you can select depend on the thresholds that have been configured in the alarm template itself. For example, if a Critical High alarm threshold of 100 Kbps is configured in the alarm template for an *Audio Bit Rate* parameter, here you will be able to select the option *Critical High (100 Kbps)* for that parameter.
```

Comment:
I would also mention the meaning of upper and lower range (i.e. this is the range specified in the protocol for the parameter), as that might not be obvious to the user